.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Report something that isn't working.
+title: ''
+assignees: ''
+
```

Comment:
```suggestion
labels: bug
```

this is what we have on android & iOS as well, to make even clearer that only bugs should be filed. but i am fine also without :)